src/main/java/core/basesyntax/services/Color.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.services;
+
+public enum Color {
+    RED("red"),
+    ORANGE("orange"),
+    YELLOW("yellow"),
+    GREEN("green"),
+    BLUE("blue"),
+    PURPLE("purple"),
+    PINK("pink"),
+    BROWN("brown"),
+    GRAY("gray"),
+    BLACK("black"),
+    WHITE("white");
+
+    private final String value;
+
+    Color(String value) {
+        this.value = value;
+    }
+
+    public static String getValue(Color color) {
+        return color.value;
+    }
```

Comment:
Let's work with enum attributes the same way as we do with class attributes - make ```getValue``` method return ```value``` of a enum entry itself

Like
```suggestion
    public String getValue() {
        return value;
    }
```
so 
```
Color.getValue(this.color)
```
can be simplified to 
```
color.getValue()
```

Reply:
Thanks for your advice, done.